sdks/js/src/index.ts

```diff
@@ -354,6 +121,41 @@ export class DustAPI {
       : this._url;
   }
 
+  async doCall(args: DoCallArgsType) {
```

Comment:
`nit`
But if we plan to expose this method publicly I'd go with `request` to align on the most common libraries.